apps/docs/src/development/packages/storybook-utils.md

```diff
@@ -51,6 +51,7 @@ Creates a default Storybook preview configuration for a project that uses `onyx`
 - Setup for dark mode (including docs page). Requires addon [`storybook-dark-mode`](https://storybook.js.org/addons/storybook-dark-mode) to be enabled in .storybook/main.ts file
 - Custom Storybook theme using onyx colors (light and dark mode)
 - Configure viewports / breakpoints as defined by onyx
+- Logs Vue emits as Storybook events
```

Comment:
```suggestion
- Logs Vue emits as Storybook actions
```

Storybook calls them [actions](https://storybook.js.org/docs/essentials/actions), should we do the same?